v1.0/docs/api-reference/definitions.md

```diff
@@ -1,5 +0,0 @@
----
```

Comment:
I just commented about this on https://github.com/GoogleCloudPlatform/kubernetes/pull/11717.
This change essentially reverts that PR.